### ISSUE TITLE
Missing descriptions of the formats

### ISSUE THREAD

**Issue by zhenlige** [PepperCode1/Continuity#525]
There should be descriptions of the formats in this repository, rather than only a reference to OptiFine (**and there is not even a link here**). Otherwise, this mod cannot be seen as a standalone mod, but must be seen as an alternative to OptiFine CTM feature.

**Comment by PepperCode1** [comment]
I will not be rewriting the OptiFine documentation when official and community-made documentation already exists publicly. I think searching "OptiFine CTM documentation" with any search engine is simple enough. I did not add any links to any external documentation because it may contain errors, but I could add some links to the wiki with a disclaimer.

**Comment by zhenlige** [comment]
If Optifine stops updating one day, this mod may become without documentation.

**Comment by zhenlige** [comment]
If Optifine stops updating one day, this mod may become without documentation.

**Comment by PepperCode1** [comment]
I don't agree. Even if OptiFine stops updating, the official and community-made documentation will not simply disappear.
